### PR TITLE
Update contributor information to new format

### PR DESCRIPTION
**Pull request by tresoldi**

### CODE REVIEW

**Comment by tresoldi** [comment]
Please check; I decided to start with WOLD because it is more complex, considering the "original authors" are actually editors.

**Review by xrotwang — APPROVED** [review]

**Comment by LinguList** [comment]
@tresoldi, you forogt the genitive, right? It is Haspelmath and Tadmor'**s** "WOLD". 

And I'd say: World Loanword Database, this is the title, not the abbreviation.

File: cldf/cldf-metadata.json [hunk 1]
```diff
@@ -11,14 +11,14 @@
     "dc:license": "https://creativecommons.org/licenses/by/3.0/",
     "dc:related": null,
     "dc:source": "sources.bib",
-    "dc:title": "The World Loanword Database",
+    "dc:title": "CLDF dataset derived from Haspelmath & Tadmor \"WOLD\" (2009)",
```
Comment (LinguList) [inline cldf/cldf-metadata.json:14]: Yes, please check with my instructions. It is AUTHOR'**s**, and WOLD is not a good title, as it is too short, and it is more explicit if one spells it out.

**Comment by tresoldi** [comment]
@LinguList on the genitive, yes. On the title, I though we were going to use shorter titles, especially when the dataset is known by that name? But I'll move to the full title and merge.

File: cldf/cldf-metadata.json (hunk 1 above)
Comment (tresoldi) [reply cldf/cldf-metadata.json:14]: Fixed with 9325f52

**Comment by xrotwang** [comment]
I find the genitive sometimes difficult - both to write (in particular for names ending in s or similar, or names including apostrophes) and read. Is "Haspelmath and Tadmor "WOLD"" really unacceptable?

**Comment by LinguList** [comment]
For me, yes. Both the WOLD, as it has a real title, and the missing
genitive *-s. alterantive is to write:


CLDF dataset derived from the "World Loanword Lexicon" by Haspelmath and
Tadmor from 2009.


But here, you will see that in one dataset, you need the article "the"
for the dataset, in one you won't need it. So we avoid fiddling with the
article "the" when using teh genitive in front. Otherwise, we can also
go for the " by "-structure, but we should decide soon.

As to dataset names: we decided long time ago to not use abbreviations,
unless they are the official name, as in "NorthEuralex". But normally,
databases have an abbreviation and a name, and we should use the name.

**Comment by xrotwang** [comment]
Ok, what about "Author et al."? "Author et al.'s"? Or "... by Author et al." required?

**Comment by LinguList** [comment]
Author et al.'s is the correct form. I have seen this at least in
publications.

**Comment by tresoldi** [comment]
`et al.'s` looks ugly to me, and I am starting to see all potential parsing problems (quotes, double quotes, ampersands, now dots...). Even the Latin genitive "aliorum" would look better.

If we are voting, I prefer the "... by X, Y et al." syntax.

**Comment by LinguList** [comment]
Please write an email thread then or open an issue on github. We are in
a place on github where not many other lexibankers read and discuss this.

You can also make a completely new proposal, but note that we have to
fix these things soon, and uglyness-factors do not necessarily decide if
it goes for regularity. My proposal was already as explicit as it can
be, there is no ampersand that is being used there. We use "and".

**Comment by xrotwang** [comment]
@tresoldi I think the single Role `Editor` would be best for Haspelmath and Tadmor here.

**Comment by tresoldi** [comment]
@xrotwang I can't believe I missed it! I'll fix it once we decide on a naming scheme, thank you.

**Pull request merged**